modules/vaos/spec/services/appointment_requests_service_spec.rb

```diff
@@ -5,13 +5,39 @@
 describe VAOS::AppointmentRequestsService do
   subject { described_class.for_user(user) }
 
-  let(:user) { build(:user, :mhv) }
-  let(:start_date) { Date.parse('2019-08-20') }
-  let(:end_date) { Date.parse('2020-08-22') }
+  let(:user) { build(:user, :vaos) }
 
   before { allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token') }
 
+  describe '#put_cancel_appointment_request' do
+    context 'when appointment_request cannot be cancelled' do
+      let(:request_body)  { build(:va_appointment_request) }
+
+      it 'returns the bad request with detail in errors' do
+        VCR.use_cassette('vaos/appointment_requests/put_cancel_request_400', match_requests_on: %i[method uri]) do
+          expect { subject.put_cancel_appointment_request(request_body) }.to raise_error(
+            Common::Exceptions::BackendServiceException
+          )
+        end
+      end
+    end
+
+    context 'when appointment_request can be cancelled' do
+      let(:request_body)  { build(:va_appointment_request) }
+
+      it 'cancels the appointment_request' do
+        VCR.use_cassette('vaos/appointment_requests/put_cancel_request', match_requests_on: %i[method uri]) do
+          response = subject.put_cancel_appointment_request(request_body)
+          expect(response).to be_an_instance_of(String).and be_empty
+        end
+      end
+    end
+  end
+
   describe '#get_requests' do
+    let(:start_date) { Date.parse('2019-08-20') }
+    let(:end_date) { Date.parse('2020-08-22') }
```

Comment:
These two values don't have a clear reason to have been added from the immediate context/code -- this method seems like it doesn't have anything to do with the rest of the changes. Was this from another commit or did something else change that required this change?

Reply:
I moved stuff in the specs around. You'd have to look at the full diff more carefully to understand the rationale for why. It's largely because your previous code review comments suggested that the descriptions for examples were not clear to you.